.github/workflows/test-wtih-vscode-build.yml

```
@@ -13,7 +13,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-20.04]
-        node-version: [14.x, 16.x]
+        node-version: [16.x]
```

```suggestion
        node-version: [16.x, 18.x]
```

Add `node18` to VsCode Build Test

---

I've already tried here. And VS Code 1.68.1 recommend that we should use the node.js with version `>=16.14.x and <17`. If we add node 18 here, buiding process will throw an warning.